src/extension.ts

```diff
@@ -43,6 +49,11 @@ export async function activate(context: ExtensionContext) {
 			return context.workspaceState.get(key, defaultValue);
 		})
 	);
+	context.subscriptions.push(
+		commands.registerCommand(vsCommands.login, async () => {
```

Comment:
so you open the browser and then what ?

Reply:
Added backlink.

Also removed `async` from the function.